Cargo.toml

```diff
@@ -455,6 +455,7 @@ members = [
 	"substrate/frame/verify-signature",
 	"substrate/frame/vesting",
 	"substrate/frame/whitelist",
+    "substrate/frame/parachain-freeze",
```

Comment:
Is this going to be deployed on the relaychain, right?  
If so, several parachain-managing pallets can be found here: https://github.com/paritytech/polkadot-sdk/tree/master/polkadot/runtime/parachains/src. However, based on the description "We need a function that can freeze a parachain" - this functionality could also be implemented as a new extrinsic+state directly within the [`Paras` pallet](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/paras/mod.rs) maybe.